src/util/kaldi-table-inl.h

```diff
@@ -340,6 +441,13 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     // or the user called FreeCurrent().. note,
     // if when called by user we are in this state,
     // it means the user called FreeCurrent().
+    kObjectLoaded, // Called NextScpLine() and found we already  yes         yes
```

Comment:
Since there is no function to test whether holder_ is empty, we need either a string like data_rxfilename_last_ (which I used before) or an extra state like this (which you asked me to add) to tell LoadCurrent whether to load the object from disk.